README.md

```diff
@@ -17,7 +17,29 @@ Log.Logger = new LoggerConfiguration()
     .Enrich.WithThreadId()
     .CreateLogger();
 ```
- 
+
+In order for the ThreadId to show up in the logging, you will need to create or modify your output template. 
```

Comment:
Would it be an improvement if we mentioned this normally applies only to sinks that use output templates? Many sinks simply include all properties without further action required, right?

> Some sinks, such as the `File` and `Console` sinks use an output template and the new `ThreadId` may not be automatically output. In that case, you will need to modify the output template.